DigitalLearningSolutions.Web/ViewModels/LearningPortal/Current/CurrentPageViewModel.cs

```diff
@@ -33,32 +19,47 @@ public class CurrentPageViewModel : BaseCoursePageViewModel
             IEnumerable<SelfAssessment> selfAssessments,
             string? bannerText,
             int page
-        ) : base(searchString, sortBy, sortDirection, bannerText, page)
+        ) : base(searchString, sortBy, sortDirection, page, bannerText)
         {
             var allItems = currentCourses.Cast<CurrentLearningItem>().ToList();
             foreach (SelfAssessment selfAssessment in selfAssessments)
             {
                 allItems.Add(selfAssessment);
             }
-            var sortedItems = SortingHelper.SortAllItems(
-                allItems,
+
+            var sortedItems = GenericSortingHelper.SortAllItems(
+                allItems.AsQueryable(),
                 sortBy,
                 sortDirection
             );
-            var filteredItems = SearchHelper.FilterLearningItems(sortedItems, SearchString).ToList();
+            var filteredItems = GenericSearchHelper.SearchItems(sortedItems, SearchString).ToList();

```

Comment:
Kept the Generic versions since there was new content merged that uses the old versions. Made the newly merged frameworks code compatible with the new helpers, but left the other new search/sort in the old helpers